extensions/gdx-setup/src/com/badlogic/gdx/setup/Dependency.java

```diff
@@ -37,4 +37,17 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if(((Dependency)obj).getName().equals(getName())) {
+            return true;
```

Comment:
If `obj` is `null` or is not an instance of `Dependency` this will crash. Could you please fix it?